### PR TITLE
Added support for passing a preferred interface

### PR DESCRIPTION
**Pull request by asbjornenge**
I needed to pass a preferred interface.

``` js
require('network-address')('wlan0')
```

:monkey: :wave:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
-  var interfaces = os.networkInterfaces()
-  for (var i in interfaces) {
-    for (var j = interfaces[i].length-1; j >= 0; j--) {
-      var face = interfaces[i][j]
-      if (!face.internal && face.family === 'IPv6') return face.address
-    }
-  }
-  return '::1'
+function reduceInterfaces(interfaces, iface) {
+    return Object.keys(interfaces).reduce((ifaces, _iface) => {
```
Comment (mafintosh) [inline index.js]: could you change this to a classic function? i still wanna support 0.10. the identation seems a bit off as well

**Comment by asbjornenge** [comment]
Like so? :smile: What a about the `cli.js`? Should we add support for it there too?

**Comment by asbjornenge** [comment]
Did that too :see_no_evil:

**Comment by mafintosh** [comment]
Great thanks :tada: I've added standard as well. Released in 1.1.0.

**Pull request merged**